packages/@webex/plugin-meetings/src/media/index.ts

```diff
@@ -19,6 +19,10 @@ import BrowserDetection from '../common/browser-detection';
 
 const {isBrowser} = BrowserDetection();
 
+export type BundlePolicy = ConstructorParameters<
```

Comment:
Ah, sorry, looks like I forgot to export MultistreamConnectionConfig when I was doing https://sqbu-github.cisco.com/WebExSquared/webrtc-media-core/pull/170 then you would be able to simply do `export type BundlePolicy = MultistreamConnectionConfig["bundlePolicy"];` but I guess your current code also works so it's fine